crypto/evp/e_aes.c

```diff
@@ -2584,14 +2584,22 @@ static int aes_gcm_ctrl(EVP_CIPHER_CTX *c, int type, int arg, void *ptr)
         memcpy(ptr, c->buf, arg);
         return 1;
 
+    case EVP_CTRL_AEAD_GET_TAGLEN:
+        return gctx->taglen <= 0 ? 16 : gctx->taglen;
+
     case EVP_CTRL_GET_IV:
-        if (gctx->iv_gen != 1 && gctx->iv_gen_rand != 1)
-            return 0;
         if (gctx->ivlen != arg)
             return 0;
         memcpy(ptr, gctx->iv, arg);
         return 1;
 
+    case EVP_CTRL_SET_IV:
+        if (gctx->ivlen != arg)
```

Comment:
If possible do this via an init() call instead.

Reply:
@slontis I need to remove those changes as I decided to wrap setting/getting ASN 1 params: https://github.com/openssl/openssl/pull/8024/files#diff-b5ce0bd68ed8ccf480748306416a7cccR85 which allows me to get rid of all the new ctrls. Still need to rebase it and fix some issue / address few other comments and should be ready for review.

Reply:
This is no longer used so marking as resolved.